lang/ui.ca.json

```diff
@@ -0,0 +1,1161 @@
+  },
+  "toolkit-error-loading": {
+    "defaultMessage": "S'ha produït un error en carregar el conjunt d'eines.",
+    "description": "Error shown if we failed to load toolkit content"
+  },
+  "toolkit-view-documentation": {
+    "defaultMessage": "Consulta la documentació de {name}",
+    "description": "Aria label for the toolkit topic right arrow"
+  },
+  "transfer-hex-message-one": {
+    "defaultMessage": "Arrossega el fitxer hexadecimal de la teva <strong>carpeta de descàrregues</strong> a la <strong>unitat MICROBIT</strong>.",
+    "description": "Text in the transfer hex dialog"
+  },
+  "transfer-hex-message-two": {
+    "defaultMessage": "Pots <strong>obrir</strong> el fitxer hexadecimal més tard per continuar editant.",
+    "description": "Text in the transfer hex dialog"
+  },
+  "transfer-hex-title": {
+    "defaultMessage": "Transfereix el fitxer hexadecimal desat a la micro:bit",
+    "description": "Title for the transfer hex dialog"
+  },
+  "try-again-action": {
+    "defaultMessage": "Torna-ho a provar",
+    "description": "Try again button text"
+  },
+  "undo": {
+    "defaultMessage": "Desfer",
+    "description": "Aria label for the undo button"
+  },
+  "unexpected-error-description": {
+    "defaultMessage": "Si us plau, torna-ho a provar o <link>raise a support request</link>  (proposa una sol·licitud d'ajuda)",
+    "description": "Text shown for unexpected error scenarios"
+  },
+  "unexpected-error-title": {
+    "defaultMessage": "Si us plau, torna-ho a provar o <link>raise a support request</link>  (proposa una sol·licitud d'ajuda)",
+    "description": "Text shown for unexpected error scenarios"
+  },
+  "untitled-project": {
+    "defaultMessage": "Projecte sense títol",
+    "description": "Title for a new project"
+  },
+  "update-firmware-action": {
+    "defaultMessage": "Actualitza microprogramari",
+    "description": "Update firmware button text"
+  },
+  "updated-change": {
+    "defaultMessage": "Fitxer actualitzat {changeName}",
+    "description": "Change made to file"
+  },
+  "visit-dot-org": {
+    "defaultMessage": "visita microbit.org (s'obre en una pestanya nova)",
+    "description": "alt text for logo link to .org"
+  },
+  "webusb-error-clear-connect-description-1": {
+    "defaultMessage": "Un altre procés està connectat a aquest dispositiu.",
+    "description": "Part of WebUSB error message"
+  },
+  "webusb-error-clear-connect-description-2": {
+    "defaultMessage": "Tanca qualsevol altra pestanya que pugui utilitzar WebUSB (per exemple, MakeCode, Python Editor) o desconnecta i torna a connectar la micro:bit abans de tornar-ho a provar.",
+    "description": "Part of WebUSB error message"
+  },
+  "webusb-error-clear-connect-title": {
+    "defaultMessage": "No es pot sol·licitar la interfície",
+    "description": "Title of error for WebUsb connection"
+  },
+  "webusb-error-default-title": {
+    "defaultMessage": "error de WebUSB",
```

Comment:
Should probably have initial capital letter.

Reply:
Requested in Crowdin